### ISSUE TITLE
Failed to configure system DNS

### ISSUE THREAD

**Issue by JustGostuY** [Kir-Antipov/outline-cli#1]
```
/usr/local/sbin/vpn: failed to configure system DNS: failed to backup DNS config file '/etc/resolv.conf.head' to '/etc/resolv.head.outlinecli.backup': rename /etc/resolv.conf.head /etc/resolv.head.outlinecli.backup: no such file or directory
```

How to fix this error on ubuntu 22.04.4 LTS?

**Comment by Kir-Antipov** [comment]
Actually, I'll leave this one open.

The problem is not really distro-dependent, it's more about your networking stack. I had the same issue on a Fedora-driven device recently. In short, the upstream CLI wrongfully expects `resolv.conf.head` to always be present on the system, which, as you can see, is not the case. It doesn't even need this file to be present at all, it just backs it up. So, this clearly is not a place to throw an error.

I'll make a PR upstream and/or add a patch for that to this repo.

**Comment by Kir-Antipov** [comment]
Should be fixed by Jigsaw-Code/outline-sdk#203.
